cmd/kube-apiserver/app/server.go

```diff
@@ -154,6 +154,9 @@ func CreateKubeAPIServer(kubeAPIServerConfig *master.Config, sharedInformers inf
 
 // CreateKubeAPIServerConfig creates all the resources for running the API server, but runs none of them
 func CreateKubeAPIServerConfig(s *options.ServerRunOptions) (*master.Config, informers.SharedInformerFactory, *kubeserver.InsecureServingInfo, error) {
+	// register all admission plugins
+	registerAllAdmissionPlugins(s.Admission.Plugins)
```

Comment:
@deads2k: my plan is to expose similar method on `genericapiserver ` and pass `s.Admission.Plugins` - just a few line below.